third_party/terraform/utils/iam.go

```diff
@@ -347,3 +348,79 @@ func compareAuditConfigs(a, b []*cloudresourcemanager.AuditConfig) bool {
 	bMap := createIamAuditConfigsMap(b)
 	return reflect.DeepEqual(aMap, bMap)
 }
+
+// Parse an import id extracting field values using the given list of regexes.
```

Comment:
eh I changed my mind from before- I think this and getDefaultValues are probably generic enough to belong in import.go instead of iam.go